test/e2e/nsx_networkinfo_test.go

```diff
@@ -262,16 +262,16 @@ func assureNamespaceDeleted(t *testing.T, ns string) {
 	return
 }
 
-func getVPCPathFromNetworkInfo(t *testing.T, ns, networkInfoName string) (networkInfo *v1alpha1.NetworkInfo) {
+func getVPCSnatIPFromNetworkInfo(t *testing.T, ns, networkInfoName string) (networkInfo *v1alpha1.NetworkInfo) {
 	deadlineCtx, deadlineCancel := context.WithTimeout(context.Background(), defaultTimeout)
 	defer deadlineCancel()
 	err := wait.PollUntilContextTimeout(deadlineCtx, 1*time.Second, defaultTimeout, false, func(ctx context.Context) (done bool, err error) {
 		networkInfo, err = testData.crdClientset.CrdV1alpha1().NetworkInfos(ns).Get(ctx, networkInfoName, v1.GetOptions{})
 		if err != nil {
-			t.Logf("Check VPC path of vpcnetworkconfigurations: %v, error: %+v", networkInfo, err)
+			t.Logf("Check default snat ip of networkinfo: %v, error: %+v", networkInfo, err)
 			return false, fmt.Errorf("error when waiting for vpcnetworkconfigurations VPC path: %s", networkInfoName)
 		}
-		if len(networkInfo.VPCs) > 0 && networkInfo.VPCs[0].VPCPath != "" {
+		if len(networkInfo.VPCs) > 0 && networkInfo.VPCs[0].DefaultSNATIP != "" {
```

Comment:
I feel it is more reasonable to check `networkInfo.VPCs[0].privateIPs` is not empty, since the consumer of this function is to read the PrivateIPs ?